scripts/swiftlint.sh

```diff
@@ -52,7 +52,7 @@ if [ ! -f "${CONFIGFILE}" ]; then
 fi 
 
 # All hunky dory, run linting
-$BINARYFILE --config "${CONFIGFILE}" --strict
```

Comment:
Will this stop the build from failing on a linting error?

Reply:
No. It will display the warnings and errors in the travisci log, but will not break the build if only warnings print. I thought that was the issue?

Reply:
I'd like to leave it as --strict. The main problem was that lint errors were being swallowed so finding what they were are a pain. If they are just warnings then it would be fine to write things like
```
public           func given(_ providerState: String)->Interaction
  {
      self.providerState = providerState
  return self
        }
```